app/views/service_worker/index.js.erb

```diff
@@ -103,6 +103,7 @@
         !url.href.includes('?preview=') && // Skip for /future.
         !url.href.includes('/delayed_job') && // Skip for Delayed Job dashboard
         !url.href.includes('/sidekiq') && // Skip for Sidekiq dashboard
+        !url.href.includes('/rails/mailers') && // Skip for mailers previews in development mode
```

Comment:
Otherwise previewing emails with http://localhost:3000/rails/mailers will dig up the shell